src/neo-vm/Slot.cs

```diff
@@ -0,0 +1,58 @@
+using Neo.VM.Types;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Neo.VM
+{
+    public class Slot : IReadOnlyList<StackItem>
```

Comment:
I removed `: IReadOnlyList<StackItem>` and it Compiles, it's needed or it will be needed in the future?

Reply:
Why remove?